app/controllers/my_facilities_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class MyFacilitiesController < AdminController
+  include DistrictFiltering
+  include Pagination
+  include SizeFiltering
+
+  before_action :authorize_my_facilities, only: %i[index ranked_facilities blood_pressure_control]
+
+  def index
+    @users_requesting_approval = policy_scope([:manage, :user, User])
+                                 .requested_sync_approval
+                                 .order(updated_at: :desc)
+
+    @users_requesting_approval = paginate(@users_requesting_approval)
```

Comment:
We could just wrap this paginate directly around the line above if you'd like. Not a big deal.